app/controllers/memberships_controller.rb

```diff
@@ -33,12 +33,11 @@ def update
     if @membership.update(membership_params)
       flash[:notice] = 'Membership Application
                         successfully updated'
-      render :show
     else
       flash[:alert] = 'A problem prevented the membership
                        application to be saved'
-      redirect_to edit_membership_path(@membership)
     end
+    redirect_to edit_membership_path(@membership)
```

Comment:
This won't work since we need to redirect to the #show page after a status update.